contracts/external/interfaces/gmx/GmxEventUtils.sol

```diff
@@ -0,0 +1,264 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.9;
+
+library EventUtils {
```

Comment:
Prefix the name with `Gmx`. 2 new-lines between contract/library name & pragma/imports